citadel/indico_citadel/util_test.py

```diff
@@ -12,16 +12,18 @@
 
 @pytest.mark.parametrize(('query', 'expected'), [
     ('title:"my event" ola person:john ola some:yes ola',
-     '+title:"my event" +person:john +(ola ola some\\:yes ola)'),
-    ('title:"my title:something"', '+title:"my title:something"'),
-    ('hello', '+(hello)'),
-    ('hey title:something', '+title:something +(hey)'),
-    ('title:something hey', '+title:something +(hey)'),
-    ('hey title:something hey person:john', '+title:something +person:john +(hey hey)'),
-    ('<*\\^()', '+(\\<*\\\\\\^\\(\\))'),
+     '+title:"my event" ola +person:john ola some\\:yes ola'),
+    ('title:"my title:something"', '+title:"my title\\:something"'),
+    ('hello   ', 'hello   '),
```

Comment:
we could `.strip()` the final string to get rid of this

Reply:
That test was meant to show it keeps the original untouched, but I don't see why we shouldn't strip it